manifests/pipelines/git-fetch-pipeline.yaml

```diff
@@ -339,21 +377,33 @@ spec:
     - test-model-rest-svc
     - stop-deployment
     workspaces:
-    - name: images-url
+    - name: images_url
       workspace: build-workspace-pv
   - name: skopeo-copy
     params:
-    - name: srcTLSverify
+    - name: SOURCE_IMAGE_URL
+      value: $(tasks.retrieve-build-image-info.source_image_url.path)
+    - name: DESTINATION_IMAGE_URL
+      value: $(tasks.retrieve-build-image-info.destination_image_url.path)
+    - name: SRC_TLS_VERIFY
+      value: "true"
+    - name: DEST_TLS_VERIFY
```

Comment:
Pipeline is failing because of this. We should use the `images_url` workspace as before. The README of that Tekton task is missing the information about workspaces but the `images_url` workspace is still there and can be used as before, see [here](https://github.com/openshift-pipelines/tektoncd-catalog/blob/5a1664f1adf1f6cc1323c6cc2a2bdf390658defb/tasks/task-skopeo-copy/0.4.1/task-skopeo-copy.yaml#L30-L33).

Reply:
@MarianMacik  I think the pipeline is failing because of this: `level=fatal msg="Invalid source name $(tasks.retrieve-build-image-info.source_image_url.path): Invalid image name \"$(tasks.retrieve-build-image-info.source_image_url.path)\", expected colon-separated transport:reference"` I updated the retrieve-build-image-info task and I think that change is causing the above error and the pipeline failure.

Reply:
@biswassri But that is the thing, we shoudn't change the way we specify images in the retrieve-build-image-info task. That should stay the same as it was before, that is using the url.txt file. Or is there any reason for this change?

Reply:
Yes, If you see the new skopeo-copy script, they are no longer user URL.txt file. The new task has added params for source and destination URLs. [here](https://github.com/openshift-pipelines/tektoncd-catalog/blob/5a1664f1adf1f6cc1323c6cc2a2bdf390658defb/tasks/task-skopeo-copy/0.4.1/task-skopeo-copy.yaml#L35-L43)

Reply:
@biswassri It still has it. [This is the latest version 0.4.1](https://github.com/openshift-pipelines/tektoncd-catalog/blob/p/tasks/task-skopeo-copy/0.4.1/task-skopeo-copy.yaml#L106). It also still has the [workspace definition and mention of copying multiple images](https://github.com/openshift-pipelines/tektoncd-catalog/blob/p/tasks/task-skopeo-copy/0.4.1/task-skopeo-copy.yaml#L29-L33).

When I decode the linked skopeo-copy.sh script, I get this:

```bash
#!/usr/bin/env bash

shopt -s inherit_errexit
set -eu -o pipefail

source "$(dirname "${BASH_SOURCE[0]}")/common.sh"
source "$(dirname "${BASH_SOURCE[0]}")/skopeo-common.sh"

# Ensure the /tekton/home/.docker directory exists
mkdir -p /workspace/home/.docker

phase "Copying '${PARAMS_SOURCE_IMAGE_URL}' into '${PARAMS_DESTINATION_IMAGE_URL}'"

set -x

if [ -n "${PARAMS_SOURCE_IMAGE_URL}" ] && [ -n "${PARAMS_DESTINATION_IMAGE_URL}" ]; then
    skopeo copy ${SKOPEO_DEBUG_FLAG} \
        --src-tls-verify="${PARAMS_SRC_TLS_VERIFY}" \
        --dest-tls-verify="${PARAMS_DEST_TLS_VERIFY}" \
        "${PARAMS_SOURCE_IMAGE_URL}" \
        "${PARAMS_DESTINATION_IMAGE_URL}"
else
    # Function to copy multiple images.
    copyimages() {
        filename="${WORKSPACES_IMAGES_URL_PATH}/url.txt"
        while IFS= read -r line || [ -n "$line" ]
        do
            cmd=""
            for url in $line
            do
                cmd="$cmd $url"
            done
            read -ra SOURCE <<<"${cmd}"
            skopeo copy "${SOURCE[@]}" --src-tls-verify="${PARAMS_SRC_TLS_VERIFY}" --dest-tls-verify="${PARAMS_DEST_TLS_VERIFY}"
            echo "$cmd"
        done < "$filename"
    }

    copyimages
fi
```
which clearly shows the possibility to use the `url.txt` file.

It is available since version 0.3.1.

Reply:
@MarianMacik okay sorry, I actually hadn't decoded the skopeo-copy script. I'll revert the retrieve info script changes. But in that case, what would we populate `SOURCE_IMAGE_URL` and `DESTINATION_IMAGE_URL` with? I thought that this was to be populated as a task result from retrieve info.

Reply:
@biswassri I see. In that case they should be empty. Otherwise the url.txt file won't be taken into consideration.